crowbar_framework/app/barclamps/zookeeper_barclamp.rb

```diff
@@ -15,20 +15,28 @@
 # limitations under the License.
 #
 
-barclamp:
-  name: apachehadoop
-  display: Hadoop framework barclamp collection
-  version: 0
-  user_managed: false
-  requires:
-    - '@crowbar'
-  member:
-    - apachehadoop
-
-crowbar:
-  layout: 1
-  order: 200
-
-nav:
-  barclamps:
-    apachehadoop: index_barclamp_path(:controller=>'apachehadoop')
+class ZookeeperBarclamp < Crowbar::Registry::Barclamp
+  name "zookeeper"
+  display "Zookeeper"
+  description "Hadoop Zookeeper"
+
+  member [
+    "apachehadoop"
+  ]
+
+  requires [
+    "hadoop"
+  ]
+
+  listed true
+
+  layout 1
+  version 0
+  schema 3
+
+  order 300
+
+  nav(
```

Comment:
Style/MethodCallParentheses: Do not use parentheses for method calls with no arguments.